extensions/extensions-helm/aissemble-configuration-store-chart/templates/webhook-serviceaccount.yaml

```diff
@@ -1,4 +1,4 @@
-{{ if .Values.webhook.serviceAccount.create }}
+{{ if and .Values.webhook.enable .Values.webhook.serviceAccount.create }}
```

Comment:
Q: I'm not familiar with this syntax.  It looks like the statement has the "and" in the wrong order, but it may just be my inexperience with helm(?)

Reply:
yeah.. helm template functions follow the syntax `functionName arg1 arg2...` and operators are implemented as functions.